api/src/api/pitch.ts

```diff
@@ -0,0 +1,165 @@
+import express, { Request, Response } from 'express';
+import { ObjectId } from 'mongodb';
+import { errorWrap } from '../middleware';
+
+import Pitch from '../models/pitch';
+
+const router = express.Router();
+
+/**
+ * Validates an ID to whether or not it is a valid MongoDB ID or not
+ * @param id Potential Pitch ID to validate
+ */
+const isValidMongoId = (id: string): boolean => ObjectId.isValid(id);
+
+// Gets all pitches
+router.get(
+  '/',
+  errorWrap(async (req: Request, res: Response) => {
+    const user = await Pitch.find({});
+    res.status(200).json({
+      message: `Successfully retrieved all pitches.`,
+      success: true,
+      result: user,
+    });
+  }),
+);
+
+// Gets pitch by pitch id
+router.get(
+  '/:pitchId',
+  errorWrap(async (req: Request, res: Response) => {
+    if (!isValidMongoId(req.params.pitchId)) {
+      res.status(400).json({
+        success: false,
+        message: 'Bad ID format',
+      });
+      return;
+    }
+
+    const pitch = await Pitch.findById(req.params.pitchId);
+    if (!pitch) {
+      res.status(404).json({
+        success: false,
+        message: 'Pitch not found with id',
+      });
+    } else {
+      res.status(200).json({
+        success: true,
+        result: pitch,
+        message: `Successfully retrieved pitch`,
+      });
+    }
+  }),
+);
+
+// Gets all pitches for a specific User
```

Comment:
This endpoint might make more sense as a User endpoint, something like `/user/:userId/pitches` - could you move this function to the `user.ts` instead?